docs/content/Coding Milestones/PoC2023q1/example1.md

```diff
@@ -33,12 +33,24 @@ they appear in this example.
    end="<!--example1-pre-kcp-end-->"
 %}
 
+{%
+   include-markdown "example1-subs/example1-space-manager.md"
+   start="<!--example1-space-manager-start-->"
+   end="<!--example1-space-manager-end-->"
+%}
+
 {%
    include-markdown "example1-subs/example1-start-kcp.md"
    start="<!--example1-start-kcp-start-->"
    end="<!--example1-start-kcp-end-->"
 %}
 
+{%
+   include-markdown "example1-subs/example1-kcp-provider.md"
```

Comment:
should it be create-kcp-provider ?

Reply:
I can change the name but it seems to long to me